src/parser/parser_utils.cpp

```diff
@@ -24,7 +24,7 @@
 namespace peloton {
 namespace parser {
 
-std::string ParserUtils::GetTableRefInfo(const TableRef* table,
+std::string ParserUtils::GetTableRefInfo(const TableRef *table,
```

Comment:
Stupid question: Is there a reason why this code shouldn't be in `TableRef::GetInfo()` instead of a separate util class?

Reply:
I guess we should ask Joy, this file was some of the older ones in this folder and carried over from the previous version of parser.